Automattic/jetpack
# Status: Introduce get_site_suffix method

Introduces a `get_site_suffix()` method in the Status package to replace all instances that try to determine the site's Calypso slug.

#### Changes proposed in this Pull Request:

* Adds globally available `get_site_suffix()` function to determine a site's Calypso slug.
* Adds basic tests for `get_site_suffix()`
* Replaces all instances of `Jetpack::build_raw_urls()` calls with new function.
* Removes private methods that do the same thing in favor of the new function.

#### Jetpack product discussion


See https://github.com/Automattic/jetpack/pull/17629#discussion_r525135593

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* We should probably test all modules/parts that had their side suffix code changed. I'll try to come up with steps.

To run unit tests:
* Follow [setup guidelines](https://github.com/Automattic/jetpack/blob/master/docs/development-environment.md#unit-testing) if you don't have unit tests set up yet.
* If you have not done so, from `/packages/status` run `composer install`.
* Run `phpunit`.

#### Proposed changelog entry for your changes:

* Introduced `get_site_suffix()` method to determine a site's Calypso slug.
